webpack/localization-plugin/src/utilities/ResxReader.ts

```diff
@@ -1,31 +1,82 @@
 // Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
 // See LICENSE in the project root for license information.
 
-import { FileSystem } from '@rushstack/node-core-library';
+import {
+  FileSystem,
+  Terminal
+} from '@rushstack/node-core-library';
 import {
   XmlDocument,
   XmlElement
 } from 'xmldoc';
+
 import {
   ILocalizedString,
   ILocalizationFile
 } from '../interfaces';
-import { ILoggingFunctions } from './Logging';
 
 const STRING_NAME_RESX: RegExp = /^[A-z_][A-z0-9_]*$/;
```

Comment:
Is there a reason we ban `$` in the names? It's a legal identifier character too.

Reply:
No good reason. I'll add that.